lib/monitoring/aws-ec2/EC2MetricFactory.ts

```diff
@@ -204,10 +228,26 @@ export class EC2MetricFactory {
   }
 
   private createMetrics(metricName: string, statistic: MetricStatistic) {
-    return this.strategy.createMetrics(
-      this.metricFactory,
-      metricName,
-      statistic
+    let metrics = new Array<string>();
+    if (metricName.startsWith("Disk")) {
```

Comment:
Perhaps have a separate `createDiskMetrics` function instead.

Reply:
That's a good suggestion, I'll do that.